doc/src/content/xdocs/spec.xml

```diff
@@ -1530,6 +1530,26 @@ void initFPTable() {
         </p>
       </section>
 
+      <section>
+        <title>Local timestamp (millisecond precision)</title>
+        <p>
+          The <code>local-timestamp-millis</code> logical type represents a timestamp in a local timezone, regardless of what specific time zone is considered local, with a precision of one millisecond.
```

Comment:
For local-timestamp-millis the semantics is specified but the interpretation of the numeric value is missing. For timestamp-millis, the interpretation of the numeric value that is specified, but an explanation of the semantics is missing. Could you please add both descriptions to both types? You can use the wording from here, if you would like to: https://github.com/apache/parquet-format/commit/fa6e468540d36122ca8a273e9c49425660c75360